R/estimate_contrasts.R

```diff
@@ -10,9 +10,49 @@
 #' comparisons. Can be one of `"holm"` (default), `"tukey"`, `"hochberg"`,
 #' `"hommel"`, `"bonferroni"`, `"BH"`, `"BY"`, `"fdr"` or `"none"`. See the
 #' p-value adjustment section in the `emmeans::test` documentation.
+#' @param effectsize Desired measure of standardized effect size, one of "none"
+#' (default), "emmeans", "marginal", or "bootES".
+#' @param bootES_type Specifies the type of effect-size measure to
```

Comment:
I would not document these in the main interface (to not alourdir down the list of arguments) and allow their change via Kwargs, and mention them within the details section

Reply:
do you want this merged before submitting the paper?

Reply:
if possible why not